### ISSUE TITLE
Improved Stability

### ISSUE THREAD

**Issue by Stefan4472** [Plain-Simple/GeometryToolkit#9]
Mainly error-handling and making the program more robust. Likely restructuring of CLI.java into more, specialized, classes.

**Comment by UnsolvedCypher** [comment]
I think we need to come up with how we want our CLIs to work. Obviously we can't put everything related to CLIs in the CLI class, so perhaps switch cases should be in a separate class or something?

**Comment by Stefan4472** [comment]
Yeah, I'm trying to figure out how to do all the processing without making
the CLI thousands of lines long... Something to bring up in our next
meeting.

**Comment by UnsolvedCypher** [comment]
I think I've found a pretty good way to do it and that's what I've done in the announcer program. I'll add it to the agenda.
